backend/Tasque.BLL/Exeptions/AwsException.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Tasque.Core.BLL.Exeptions
+{
+    public class AwsException : Exception
```

Comment:
As we will use more than one aws service, the name of exception is not the best)